pyHS100/discover.py

```diff
@@ -61,26 +61,26 @@ def discover(protocol: TPLinkSmartHomeProtocol = None,
         return devices
 
     @staticmethod
-    def discover_single(ip_address: str,
+    def discover_single(host: str,
                         protocol: TPLinkSmartHomeProtocol = None
                         ) -> SmartDevice:
         """
         Similar to discover(), except only return device object for single
-        passed device given by IP address.
+        passed device given by host name.
```

Comment:
As IP addresses are still accepted, how about "Except only return device object for a single host"?

Reply:
>As IP addresses are still accepted, how about "Except only return device object for a single host"?

Could someone elaborate on this? I use pyHS100 on a nat'ed LAN without using a name server. I specify ip address. I've never used a DNS name.

Reply:
It doesn't change anything in that case, the main point of this PR is to use socket's `create_connection` to allow also passing hostnames instead of IP addresses. I just like to see that the original way of things remains intact (like @khenriks did by not removing `PYHS100_IP` and `--ip`, merely adding a host option besides them, which is great!).

Reply:
Alright, that's done.

@jhalbrecht, just to clarify, this will still work with IP addresses, but it will also work with host names for people who want to use those.